docs/DeveloperGuide.md

```diff
@@ -1210,12 +1210,45 @@ testers are expected to do more *exploratory* testing.
     - **Rationale:** Keying in a wrong prefix is a fairly common user mistake and the existing error message does not seem to match the actual error happening. Detecting prefix may restrict users from typing inputs with prefix such as `n/` or `a/` as parameter string inputs, hence we will need to add the functionality to do it as well.
 
+
+3. **Improve error messages related to handling of index**
+    - **Problem:** Currently, having an invalid value (e.g. `0`, `1 2`) for `CUSTOMER_INDEX` and `ORDER_INDEX` will result in an error stating "Invalid command format!" along with the command format. While in the command format it states that said parameters must be a positive integer, the error message could be more concise.
+    - **Example:** User tries to delete a customer using `deletec 0`, the following error message is displayed:
+      ```
+      Invalid command format! 
+      deletecustomer: Deletes the customer identified by the index number used in the displayed customer list.
+      Parameters: CUSTOMER_INDEX (must be a positive integer)
+      Example: deletecustomer 1
+      ```
+    - **Solution:** Shorten the error message to state that the index should be a positive integer.
+    - **Rationale:** Stating that the command format is invalid in the first line of the error message may confuse the user in a case where they type an invalid index such as `1a` by mistake, as it is only the parameters which are incorrect in this case. Making the error message more concise also provides users with a better experience.
+
+
+4. **Improve display of longer messages or entries**
+   - **Problem:** Currently, longer messages or entries can be cut off, especially for users with smaller resolution/larger scale.
+   - **Example:** ![longMessages](images/longMessages.png)
+   - **Solution:** Improve text wrapping and allow customisable/resizeable GUI.
+   - **Rationale:** Longer messages/entries being cut off may be inconvenient for certain users (e.g. being required to scroll left/right to read an error message). Hence, providing text wrapping, allowing users to customise the GUI (e.g. whether they prefer to display an ellipsis, wrap text, etc.) and resize the GUI will give them more freedom to adjust based on their preferences and for convenience.
+
+
+5. **Include a warning if there is only one domain label provided for email**
+   - **Problem:** Currently, the `EMAIL` parameter accepts emails with only one domain label e.g. `johndoe@gmail`
+   - **Solution:** Allow emails with a single domain label, but provide a warning for it.
+   - **Rationale:** It is possible for some email address to have only one domain label, but we deem it unlikely to happen for the target users of this application. It is more likely that the user may forget to include a second domain label (e.g. missing `.com` in `johndoe@gmail.com`).
```

Comment:
```suggestion
   - **Rationale:** It is possible for some email address to have only one domain label, but we deem it unlikely to happen for the target users of this application. It is more likely that the user may have forgotten to include a second domain label (e.g. missing `.com` in `johndoe@gmail.com`).
```

Reply:
not too sure about this one

Reply:
"may have forgotten" is more or talking about something that has happened, so i don't think it's really correct

Reply:
but aren't we saying that when they put one domain label referring to previous sentence so it kind of happened

Reply:
but small issue if u think it is ok just merge